config.py

```diff
@@ -1201,6 +1201,26 @@
 confidence interval of the mean decoding score.
 """
 
+csp_reg: float = 0.1
+"""
+Regularization used in the covariance estimator when calculating CSPs.
+"""
+
+cluster_stats_alpha: float = 0.05
+"""
+Statistic level used in the time frequency script.
+"""
+
+cluster_stats_alpha_t_test: float = 0.1
+"""
+Threshold level used to determine clusters.
+"""
+
+n_permutations: int = 10_000
```

Comment:
Ah cool, I'd forgotten about this way to make large numbers more readable! 👌

Reply:
yep, this is Alexandre's idea!